eeps/eep-0069.md

```diff
@@ -1,4 +1,4 @@
-    Author: Isabell Huang <isabell.huang@ericsson.com>
+    Author: Isabell Huang <isabell(at)erlang(dot)org>
     Status: Accepted
     Type: Standards Track
```

Comment:
I guess we should add the Erlang-Version when this gets merged in OTP-28?

Reply:
yes, when changing the status to final.